### PR TITLE
Remove unnecessary method closed?

### PR DESCRIPTION
**Pull request by siddharth28**
`state_machine` provides this following boolean method.

Also did not remove specs so that the functioning of this method can be verified.

### CODE REVIEW

File: core/app/models/spree/adjustment.rb [hunk 1]
```diff
@@ -73,10 +73,6 @@ class Adjustment < Spree::Base
     extend DisplayMoney
     money_methods :amount
 
-    def closed?
```
Comment (nishant-cyro) [inline core/app/models/spree/adjustment.rb]: @siddharth28 - We are using this method here - https://github.com/siddharth28/spree/blob/master/core/app/models/spree/adjustment.rb#L88.

Comment (nishant-cyro) [inline core/app/models/spree/adjustment.rb]: @siddharth28 - Or if there is something I am missing, can you provide a description?

**Comment by siddharth28** [comment]
@nishant-cyro added description.

**Comment by nishant-cyro** [comment]
:+1:

**Comment by priyank-gupta** [comment]
:+1:

**Pull request merged**